Makefile

```diff
@@ -0,0 +1,17 @@
+# Makefile based on https://www.cs.colby.edu/maxwell/courses/tutorials/maketutor/
+
+CC=cc
+CFLAGS=-I.
```

Comment:
what does this flag do? we should probably add a comment for it. Also why is there a period at the end?

Reply:
the flag tells the compiler to look in the current directory (`.`) for files to Include. I got it from the tutorial which I linked at the top of the makefile--I will comment each line with details so we don't have to go to a link to find out.